docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1004,6 +1004,25 @@ You can use the [`collapse-rename-collapsedheight` codemod](https://github.com/m
 
   You can use the [`use-transitionprops` codemod](https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-codemod#use-transitionprops) for automatic migration.
 
+- Change the default value of `anchorOrigin.vertical` to follow the Material Design guidelines. The menu now displays below the anchor instead of on top of it.
+  You can restore the previous behavior with:
+
+  ```diff
+   <Menu
+  +  anchorOrigin={{
+  +    vertical: 'top',
+  +    horizontal: 'left',
+  +  }}
+  ```
+
+- The `MenuItem` component inherits the `ButtonBase` component instead of `ListItem`.
+  The class names related to "MuiListItem-\*" are removed and theming `ListItem` is no longer affecting `MenuItem`.
+
+  ```diff
+  -<li className="MuiButtonBase-root MuiMenuItem-root MuiListItem-root">
+  +<li className="MuiButtonBase-root MuiMenuItem-root">
```

Comment:
Let's also add this BC:

![image](https://user-images.githubusercontent.com/4512430/122163704-e4cbf780-ce75-11eb-8a44-dfefee4b89d5.png)

And please update the PR description as well :)